code_schemes/solve_problems.json

```diff
@@ -3,7 +3,118 @@
   "Name": "Solve Problems",
   "Version": "0.0.0.1",
   "Codes": [
-
+    {
+      "CodeID": "code-40e87286",
+      "CodeType": "Normal",
+      "DisplayText": "national government",
+      "NumericValue": 1,
+      "StringValue": "national_government",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-d9b5155c",
+      "CodeType": "Normal",
+      "DisplayText": "local government",
+      "NumericValue": 2,
+      "StringValue": "local_government",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-2f981510",
+      "CodeType": "Normal",
+      "DisplayText": "experts/scholars",
+      "NumericValue": 3,
+      "StringValue": "experts_scholars",
```

Comment:
```suggestion
      "StringValue": "experts_or_scholars",
```

Reply:
Thanks - changed to experts_and_scholars after consultation with RDA